scripts/docker/build/build-image_x86-64.sh

```diff
@@ -11,7 +11,7 @@ OPENSSL_VERSION=$(docker run --rm -it $image:latest-$arch sh -c "cat \${OPENSSL_
 
 OS_VERSION=$(docker run --rm -it $image:latest-$arch cat /etc/alpine-release | grep -Po "\d+\.\d+")
 
-IMAGE_TAG=${OS_VERSION}-gcc${GCC_VERSION}-openssl${OPENSSL_VERSION}-$(git rev-parse --short HEAD)-$arch
+IMAGE_TAG=${OS_VERSION}-gcc${GCC_VERSION}-openssl${OPENSSL_VERSION}-${CIRCLE_SHA1::8}-$arch
```

Comment:
Can you please set `${CIRCLE_SHA1::8}` to `$(git rev-parse --short HEAD)` in case it's empty (a local run is done)?